tests/cpu_bugs/xen_grub_setup.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+
+use warnings;
+use strict;
+use base "opensusebasetest";
+use testapi;
+use utils;
+use bootloader_setup 'replace_grub_cmdline_settings';
+
+sub run {
+    my $self  = shift;
+
+    replace_grub_cmdline_settings('unit=0', 'unit=1', update_grub => 1, search => '^GRUB_SERIAL_COMMAND=');
```

Comment:
Can the 'unit=1" work for each machine? whatever serial console is COM1 or COM2, right?

Reply:
1. For our 3 machine in XEN testing, they are all COM2.
2. If testing on COM1 machine, we no need do anything on grub. So we can skip test on COM1 machine, or else don't care whether do the test. Because this test will no effect on COM1 machine.

Reply:
> Can the 'unit=1" work for each machine? whatever serial console is COM1 or COM2, right?

1. For our 3 machine in XEN testing, they are all COM2.
2. If testing on COM1 machine, we no need do anything on grub. So we can skip test on COM1 machine, or else don't care whether do the test. Because this test will no effect on COM1 machine.

Reply:
1. For our 3 machine in XEN testing, they are all COM2.
2. If testing on COM1 machine, we no need do anything on grub. So we can skip test on COM1 machine, or else don't care whether do the test. Because this test will no effect on COM1 machine.